front-end/src/components/forms/TeamMemberForm.js

```diff
@@ -92,17 +114,69 @@ class TeamMemberForm extends React.Component {
               </Checkbox>
             </FormGroup>
 
-            <SelectTeam name="team" teams={this.props.teams} selected={this.state.team} onSelect={(e) => {
-              this.onInputChange(e);
-            }} />
+            <ControlLabel>Experience</ControlLabel>
+            <FormGroup controlId="experience">
+              <Radio
```

Comment:
This group of radio buttons could be generated in a loop.  That would cut down on the lines of code and code reuse